Makefile

```diff
@@ -220,6 +220,163 @@ ubuntu_docker_build: Dockerfile-ubuntu.$(PLATFORM).tag
 ubuntu_docker_publish: Dockerfile-ubuntu.$(PLATFORM).tag
 	docker push $(CONTAINER_NAME_TAG)
 
+ARCH_NAME ?= $(shell uname -m)
+DOCKER_IMAGE_TAG ?= 0.3.2-PR-996.3
+CENTOS_DOCKER_IMAGE_NAME ?= docker.h2o.ai/opsh2oai/datatable-build-$(ARCH_NAME)_centos7:$(DOCKER_IMAGE_TAG)
+UBUNTU_DOCKER_IMAGE_NAME ?= docker.h2o.ai/opsh2oai/datatable-build-$(ARCH_NAME)_ubuntu:$(DOCKER_IMAGE_TAG)
```

Comment:
Do we still need Ubuntu here?

Reply:
We need ubuntu for coverage. Also I think we might test on Ubuntu, since it's widely used and the tests are quite fast.